test/runtests.jl

```diff
@@ -13,17 +24,21 @@ using StaticArrays
 using Statistics
 using Test
 
+@nospecialize
+
+if isdefined(Base, :Experimental) && isdefined(Base.Experimental, Symbol("@compiler_options"))
+    @eval Base.Experimental.@compiler_options compile=min optimize=0 infer=no
```

Comment:
May I ask to this line how this works? I am trying to use this in my code and it seems to not work (compared to starting Julia with those settings as command line parameters). Is there a trick to make this work?

Reply:
My approach is https://github.com/JuliaManifolds/Manifolds.jl/pull/507/files